mkt/reviewers/views.py

```diff
@@ -992,43 +993,11 @@ class ReviewersSearchView(SearchView):
     authentication_classes = [RestSharedSecretAuthentication,
                               RestOAuthAuthentication]
     permission_classes = [GroupPermission('Apps', 'Review')]
+    filter_backends = [SearchQueryFilter, ReviewerSearchFormFilter,
+                       SortingFilter]
     form_class = ApiReviewersSearchForm
     serializer_class = ReviewersESAppSerializer
-
-    def search(self, request):
-        # Parse form.
-        form = self.form_class(request.GET if request else None)
-        if not form.is_valid():
-            raise form_errors(form)
-        form_data = form.cleaned_data
-
-        # Status filter.
-        data = search_form_to_es_fields(form_data)
-        if form_data.get('status') != 'any':
-            data.update(status=form_data.get('status'))
-
-        # Do filter.
-        sq = apply_reviewer_filters(request, WebappIndexer.search(),
-                                    data=form_data)
-        sq = WebappIndexer.get_app_filter(request, data, sq=sq, no_filter=True)
-
-        page = self.paginate_queryset(sq)
-        return self.get_pagination_serializer(page), request.GET.get('q', '')
-
-
-def apply_reviewer_filters(request, qs, data=None):
-    for k in ('has_info_request', 'has_editor_comment'):
-        if data.get(k) is not None:
-            qs = qs.filter('term', **{'latest_version.%s' % k: data[k]})
-    if data.get('is_escalated') is not None:
-        qs = qs.filter('term', is_escalated=data['is_escalated'])
-    is_tarako = data.get('is_tarako')
-    if is_tarako is not None:
-        if is_tarako:
-            qs = qs.filter('term', tags='tarako')
-        else:
-            qs = qs.filter(~F('term', tags='tarako'))
-    return qs
+    paginator_class = ESPaginator
```

Comment:
Do you really need this? It is already defined to the same value by the base class, `SearchView`.